lib/migration.js

```diff
@@ -111,7 +115,7 @@ function mixinMigration(MySQL, mysql) {
     });
   };
 
-  MySQL.prototype.alterTable = function(model, actualFields, actualIndexes, done, checkOnly) {
+  MySQL.prototype.alterTable = function(model, actualFields, actualIndexes, actualFks, done, checkOnly) {
```

Comment:
Since this is changing (semi)public API exposed by the mysql connector, should we keep supporting the old method signature `alterTable(model, actualFields, actualIndexes, done, checkOnly)` for backwards compatibility?